packages/libs/components/src/map/BoundsDriftMarker.tsx

```diff
@@ -281,30 +281,30 @@ export default function BoundsDriftMarker({
   };
 
   // add click events for highlighting markers
-  const handleClick = (e: LeafletMouseEvent) => {
-    // check the number of mouse click and enable function for single click only
-    if (e.originalEvent.detail === 1) {
-      if (setSelectedMarkers) {
-        if (selectedMarkers?.find((id) => id === props.id)) {
-          setSelectedMarkers((prevSelectedMarkers: string[]) =>
-            prevSelectedMarkers.filter((id: string) => id !== props.id)
-          );
-        } else {
-          // select
-          setSelectedMarkers((prevSelectedMarkers: string[]) => [
-            ...prevSelectedMarkers,
-            props.id,
-          ]);
+  const handleClick = useCallback(
```

Comment:
useCallback doesn't seem essential but seems sensible given it is debounced and this may avoid the debouncing being reset unnecessarily.